src/ComponentCell.js

```diff
@@ -26,7 +26,9 @@ export default class ComponentCell extends PureComponent {
       onMouseDown, onMouseOver, onDoubleClick, onContextMenu, attributes
     } = this.props
     const style = { width }
-
+    const component = forceComponent
+      ? React.cloneElement(this.props.component, {editing})
+      : this.props.component
```

Comment:
What is this achieving exactly? The editing value is only for the classname

Reply:
## Steps to reproduce:
1. Click on any Header cell.
2. Try to navigate to Input Cell using keyboard and press enter or start typing. 

You wont be able to type, unless you click on the input element.

## Reproduction code:
```javascript
import React from "react";
import ReactDOM from "react-dom";
import Datasheet from 'react-datasheet';
import "react-datasheet/lib/react-datasheet.css";

export default class BasicSheet extends React.Component {
  constructor (props) {
    super(props)
    this.state = {
      value: "",
      grid: []
    }
  }

  componentWillMount() {
    const component = (
      <input type="text" />
    );
    this.setState({
      grid: [
        [
          {readOnly: true, value: ''},
          {value: 'A', readOnly: true},
          {value: 'B', readOnly: true}
        ],
        [
          {readOnly: true, value: 1},
          {component: component, forceComponent: true},
          {component: component, forceComponent: true}
        ],
        [
          {readOnly: true, value: 2},
          {component: component, forceComponent: true},
          {component: component, forceComponent: true}
        ],
        [
          {readOnly: true, value: 3},
          {component: component, forceComponent: true},
          {component: component, forceComponent: true}
        ]
      ]
    })
  }


  render () {
    return (
      <Datasheet
        data={this.state.grid}
        valueRenderer={(cell) => cell.value}
        onContextMenu={(e, cell, i, j) => cell.readOnly ? e.preventDefault() : null}
      />
    )
  }
}
```